README.md

```diff
@@ -1 +1,8 @@
-# myre2
\ No newline at end of file
+# myre2
+
+
+def myfunc(lis):
+    lis.sort()
+    return lis[-1]
```

Comment:
Always follow python3 standard coding guidelines. 

Blank line after every block of code